lizard.py

```diff
@@ -41,11 +42,10 @@
 except ImportError:
     pass
 
-
 VERSION = "1.10.4"
 
-DEFAULT_CCN_THRESHOLD, DEFAULT_WHITELIST, \
-    DEFAULT_MAX_FUNC_LENGTH = 15, "whitelizard.txt", 1000
+DEFAULT_CCN_THRESHOLD, DEFAULT_ND_THRESHOLD, DEFAULT_WHITELIST, \
+    DEFAULT_MAX_FUNC_LENGTH = 15, 7, "whitelizard.txt", 1000
```

Comment:
@mehrdad89 instead of changing directly in lizard.py and clike.py, I'd like these changes to be in an extension first (see lizard_ext/lizardexitcount.py for example). Then use command like:

    lizard -E nd

To include the new metrics. We can have the metrics in the default result collection when it's stabilised later.
You can make the change, or I can accept your pull request and make the change for you. What do you prefer?